docs/core/extensions/logging-providers.md

```diff
@@ -3,17 +3,18 @@ title: Logging providers
 description: Learn how the logging provider API is used in .NET applications.
 author: IEvangelist
 ms.author: dapine
-ms.date: 03/13/2023
+ms.date: 06/23/2023
 ---
 
 # Logging providers in .NET
 
 Logging providers persist logs, except for the `Console` provider, which only displays logs as standard output. For example, the Azure Application Insights provider stores logs in Azure Application Insights. Multiple providers can be enabled.
 
 The default .NET Worker app templates:
+CreateApplicationBuilder
```

Comment:
This looks out of place here -- it's not part of the list or the intro sentence for the list.

Reply:
It is, good catch!